src/applications/financial-status-report/wizard/DelayedLiveRegion.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+/* aria-live region must be on the screen FIRST when rendering conditional content
+   artificial delay on the content so it will always render second and the region will announce the updated conditional content
+   not wrapping the wizard as a whole to avoid a overly verbose experience on screen readers */
+const DelayedLiveRegion = ({ children, delay = 100 }) => {
+  const [timer, setTimer] = useState(0);
+  const [showContent, setShowContent] = useState(false);
+  const shouldRenderContent = showContent;
+
+  useEffect(
+    () => {
+      let interval = null;
+
+      if (interval === null) {
+        interval = setInterval(() => {
+          setTimer(milliseconds => milliseconds + 1);
+        }, 1);
+      }
+
+      if (delay + 1 === timer) {
+        setShowContent(true);
+        clearInterval(interval);
+      }
+
+      return () => {
+        clearInterval(interval);
+        setShowContent(false);
+      };
+    },
+    [timer, delay],
+  );
+
+  return (
+    <section aria-live="polite">{shouldRenderContent && children}</section>
```

Comment:
Awesome, so if I'm guessing right, this would return the light grey box as a `<section>` with `aria-live="polite"` 👍  Some attributes that could further enhance this experience we could append to the light grey box `<section>` include:

- add in `aria-atomic="true"`, to ensure the full section is read out
- if possible convert the `p` in the results that says "This isn't the form you need..." to a `h2` with a unique `id` and add in an `aria-labelledby` that points to that `h2` using `aria-labelledby="unique-id"`; this will help screen reader users hear the bit "This isn't the form you need" whenever they tab into a link or button in that region.
- if possible add the attribute `role="region"` so it's discoverable on the elements list for VO users and other screen readers

Reply:
Look like the <section/> element has implicit role of region so I am gonna leave that one off

![image](https://user-images.githubusercontent.com/23741323/115285646-1ba6bb00-a11c-11eb-8886-6f466e41162b.png)

Reply:
Added all requested edits including heading hierarchy by adding the h2 element